script.js

```diff
@@ -1,4 +1,425 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+//Roll Dice Function
+var rollDice = function () {
+  // produces a decimal between 0 and 6
+  var randomDecimal = Math.random() * 6;
+  // take off the decimal
+  var randomInteger = Math.floor(randomDecimal);
+  // it's a number from 0 - 5 ... add 1
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+// Define Game Modes
+var startGame = `displaying game modes`;
+var selectGameModes = `select game mode`;
+// 2 Player normal
+var twoPlayer_EnterDice = `enter dice roll`;
```

Comment:
For the naming of variables, stick to camelCase (i.e. without the underscore) due to conventions. Underscores are reserved for constants which we will not learn in this course.